### PR TITLE
COV 679: Add fields to collapsed view

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: higher_health/forms.py [hunk 1]
```diff
@@ -88,22 +88,24 @@ class HealthCheckQuestionnaire(forms.Form):
...
     )
     facility_destination_university = forms.ModelChoiceField(
-        label="Please select an institution",
+        label="Please select an institution:",
         queryset=models.University.objects.all(),
         required=False,
     )
     facility_destination_university_other = forms.CharField(
-        label="Other", required=False
+        label="Enter a name of an institution:", required=False
```
Comment (miltontony) [inline higher_health/forms.py]: Shouldn't this be "Enter *the* name of *the* institution:"? (similarly for the campus question)
Please check with SxD on the wording

**Review by miltontony — APPROVED** [review]
Small comment, otherwise looks good.

File: higher_health/templates/healthcheck_questionnaire.html [hunk 2]
```diff
@@ -527,7 +530,13 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
                   excerpt.innerHTML =  '<h5>'+entryInfo.entryName + ' '+ entryInfo.entrySurname + ' | ' + entryInfo.entryAge() + ' | ' + entryInfo.entryAddress+'</h5>';
                 }
                 else if(collapsiblesToggle === 'step_2') {
-                  excerpt.innerHTML = '<h5>'+ entryInfo.entryUni() +' | '+entryInfo.entryCampus()+'</h5>';
+                  if(entryInfo.entryUni() === 'Other' || entryInfo.entryCampus() === 'Other') {
+                    excerpt.innerHTML = '<h5>'+ entryInfo.entryUniOther +' | '+ entryInfo.entryCampusOther +'</h5>';
+                    console.log('True');
```
Comment (rudigiesler) [inline higher_health/templates/healthcheck_questionnaire.html]: Are we keeping these console logs in to test in QA, and removing before production?

Comment (Mitso) [reply higher_health/templates/healthcheck_questionnaire.html]: Good catch! Thanks @rudigiesler. Removing...

**Pull request merged**